docs/reference/projectyml.rst

```diff
@@ -65,29 +65,42 @@ This section explains how to configure a project-wide pip index. This index will
 modules' dependencies.
 By default, a project created using the :ref:`project-creation-guide` is configured to install packages from ``https://pypi.org/simple/``.
 The :class:`~inmanta.module.ProjectPipConfig` section of the project.yml file offers options to configure this behaviour.
+Some of these options are detailed below:
 
 pip.use-system-config
 """""""""""""""""""""
 
-This option determines the isolation level of the project's pip config. When false, any pip config set on the system
+This option determines the isolation level of the project's pip config. When false (the default), any pip config set on the system
```

Comment:
> any pip config ...

May need to be rephrased depending on the outcome of [this discussion](https://github.com/inmanta/inmanta-core/pull/6655#discussion_r1386279542).